perfzero/lib/perfzero/perfzero_config.py

```diff
@@ -76,6 +76,16 @@ def add_setup_parser_arguments(parser):
       type=str,
       help='The docker tag to use if building a docker image.'
       )
+  parser.add_argument(
+      '--include_dirs_in_site_package',
```

Comment:
I am wondering if it would be better to name this config `--site_package_downloads`. Similar to `--data_downloads`, `--site_package_downloads` can accept different protocols such as file://, gcs:// or git://. And PerfZero can copy/download/checkout data to docker's site package dir based on the user-specified protocol. `include_dirs_in_site_package` sounds like a verb but not noun.

Reply:
Done, nice suggestion